test/powershell/engine/Remoting/ImplicitRemotingBatching.Tests.ps1

```diff
@@ -0,0 +1,107 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "TestImplicitRemotingBatching hook should correctly batch simple remote command pipelines" -Tags 'Feature','RequireAdminOnWindows' {
+
+    BeforeAll {
+
+        if ($isWindows)
```

Comment:
if you wanted to eliminate the indentation, you could do
```
BeforeAll {
    if ( ! $IsWindows ) {
        return
    }
. . .
```

Reply:
Done